spark-bigquery-python-lib/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.spark</groupId>
+    <artifactId>spark-bigquery-parent</artifactId>
+    <version>0.30.0-SNAPSHOT</version>
+    <relativePath>../spark-bigquery-parent</relativePath>
+  </parent>
+
+  <artifactId>spark-bigquery-support</artifactId>
+  <name>PySpark-BigQuery support library</name>
```

Comment:
should we add version tag to the python lib?

Reply:
It gets it from its parent. See installed version:
```
$ ls -l ~/.m2/repository/com/google/cloud/spark/spark-bigquery-support/0.30.0-SNAPSHOT/
total 16
-rw-r--r-- 1 *** ***  230 Jul 18 22:17 _remote.repositories
-rw-r--r-- 1 *** ***  733 Jul 18 22:17 maven-metadata-local.xml
-rw-r--r-- 1 *** *** 1049 Jul  9 19:14 spark-bigquery-support-0.30.0-SNAPSHOT.pom
-rw-r--r-- 1 *** *** 1684 Jul 18 22:17 spark-bigquery-support-0.30.0-SNAPSHOT.zip
```